app/src/androidTest/java/com/example/bookwormadventuresdeluxe2/LoginActivityTest.java

```diff
@@ -22,18 +18,19 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
+import static com.example.bookwormadventuresdeluxe2.TestUtils.signIn;
+import static com.example.bookwormadventuresdeluxe2.TestUtils.signOut;
 
 /**
- * Tests for login screen, cannot be
+ * Tests for login screen
```

Comment:
* Tests for login screen, cannot be run more than 6 times in 5 minutes
 * or device will be blocked for too many login attempts

I didnt finish the sentence. Its in create account errors though lol

Reply:
My only question is
![image](https://user-images.githubusercontent.com/39696855/97821063-3d93aa80-1c6e-11eb-8d0b-6e925eb323c6.png)

Do you need to waitForText "Profile" or does clickOnText have it's own 5 second time out? Because I thought "Profile" might not be found instantly

Reply:
I cant find anything on clickOnText having a timer. But it should be good because its not relying on firebase but the app resources!

Reply:
we shouldn't need to waitForText Profile because a precondition to this test is that it's already visible